main.h

```diff
@@ -22,5 +22,6 @@ int handle_char(va_list args);
 int handle_integer(va_list args);
 int handle_string(va_list args);
 int handle_format_specifier(const char *fmt, size_t *index, va_list args);
```

Comment:
@tingru0515 : I think these `handle...` functions are not being used anymore too. So they could be deleted.